src/theory/bv/theory_bv_rewrite_rules_normalization.h

```diff
@@ -1451,6 +1451,83 @@ inline Node RewriteRule<BitwiseSlicing>::apply(TNode node)
   return result;
 }
 
+template <>
+inline bool RewriteRule<NormalizeEqPlusNeg>::applies(TNode node)
+{
+  return node.getKind() == kind::EQUAL
+         && (node[0].getKind() == kind::BITVECTOR_PLUS
+             || node[1].getKind() == kind::BITVECTOR_PLUS);
+}
+
+template <>
+inline Node RewriteRule<NormalizeEqPlusNeg>::apply(TNode node)
+{
+  Debug("bv-rewrite") << "RewriteRule<>(" << node << ")" << std::endl;
```

Comment:
Nit: This should probably say `RewriteRule<NormalizeEqPlusNeg>`.

Reply:
Done.